### PR TITLE
React: Add empty row button for data entry table

### LINKED ISSUE #159: Add participants: "Add empty row" action button as 'lip' at bottom-center of table
Task points: 2.5

### PR DESCRIPTION
**Pull request by annnag**
Closes #159

### CODE REVIEW

File: react/src/pages/utils/upload/DataEntryTable.tsx [hunk 1]
```diff
@@ -232,6 +235,22 @@ export default function DataEntryTable(props: DataEntryTableProps) {
...
+                            <TableCell className={classes.buttonCell} colSpan={100}>
+                                <Button
+                                    className={classes.newRowButton}
+                                    variant="contained"
+                                    color="default"
+                                    disableElevation
+                                    startIcon={<Add />}
+                                    onClick={event => {
+                                        setRows(rows.concat(createEmptyRows(1)));
+                                    }}
```
Comment (kevinlul) [inline react/src/pages/utils/upload/DataEntryTable.tsx:248]:
```suggestion
                                    onClick={() => setRows(rows.concat(createEmptyRows(1)))}
```

**Review by kevinlul — CHANGES_REQUESTED** [review]
It feels a little laggy on my end as the action only happens after the animation moves through the entire width of the button.

**Comment by annnag** [comment]
Is it better now?@kevinlul

**Comment by kevinlul** [comment]
Yeah, it's a lot snappier.

**Review by kevinlul — APPROVED** [review]

**Review by frewmack — APPROVED** [review]

**Pull request merged**